### PR TITLE
Update Supported models section

### PR DESCRIPTION
**Pull request by jmayank1511**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +87,65 @@ For additional information and usage, run:
...
+    <td rowspan="2">Punctuation and Capitalization</td>
+    <td>Bert-Base</td>
+    <td rowspan="2">nemo2riva --onnx-opset 18 &lt;path to .nemo model&gt;</td>
+    <tr>
+    <td>Bert-Large</td>
+    </tr>
+    <tr>
+    <td>Neural Machine Translation</td>
+    <td>Megatron</td>
+    <td>nemo2riva &lt;path to .nemo model&gt; </td>
```
Comment (rmittal-github) [inline README.md:149]: should we include `--key` in the nemo2riva commands?

Comment (rmittal-github) [reply README.md:149]: also, should we also add `--out` explicitly so user can specify name of the riva model to be generated?

File: README.md [hunk 2]
```diff
@@ -87,3 +87,65 @@ For additional information and usage, run:
...
+    </tr>
+    <tr>
+    <td rowspan="2">Punctuation and Capitalization</td>
+    <td>Bert-Base</td>
+    <td rowspan="2">nemo2riva --onnx-opset 18 &lt;path to .nemo model&gt;</td>
+    <tr>
+    <td>Bert-Large</td>
+    </tr>
+    <tr>
+    <td>Neural Machine Translation</td>
```
Comment (rmittal-github) [inline README.md:147]: for nmt, need to add `--max-dim=1024`

Comment (jmayank1511) [reply README.md:148]: Done

File: README.md (hunk 1 above)
Comment (jmayank1511) [reply README.md]: Done

**Review by rmittal-github — APPROVED** [review]

**Comment by mehadi92** [comment]
Hi @jmayank1511, @rmittal-github  does it support [parakeet-tdt_ctc](https://huggingface.co/nvidia/parakeet-tdt_ctc-0.6b-ja) model also can you please list supported diarizer model - Thanks

**Comment by jmayank1511** [comment]
You need to extract the decoder first before passing to nemo2riva. 

Supported Diarization model : TitaNet.

**Comment by mehadi92** [comment]
@jmayank1511 

"You need to extract the decoder first before passing to nemo2riva." 

for [parakeet-tdt_ctc](https://huggingface.co/nvidia/parakeet-tdt_ctc-0.6b-ja) model? 
Is there any script for it? 

"Supported Diarization model : TitaNet." 

Mentioning this in [supported-models](https://github.com/nvidia-riva/nemo2riva?tab=readme-ov-file#supported-models) will be good

**Pull request merged**